waspc/data/Generator/templates/react-app/src/auth/useAuth.js

```diff
@@ -18,4 +18,4 @@ async function getMe() {
   }
 }
 
-getMe.queryCacheKey = 'auth/me'
+getMe.queryCacheKey = ['auth/me']
```

Comment:
Reference for this and all other array-related changes: https://tanstack.com/query/v4/docs/guides/migrating-to-react-query-4#query-keys-and-mutation-keys-need-to-be-an-array